jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantWidget/ExtendedVariantWidget.js

```diff
@@ -31,71 +33,323 @@ export default jbrowse => {
     const { FeatureDetails, BaseFeatureDetails, BaseCard } = jbrowse.jbrequire(
       '@jbrowse/core/BaseFeatureWidget/BaseFeatureDetail',
     )
+    var useStyles = styles;
 
-    const useStyles = makeStyles(() => ({
-        table: {
-            padding: 0,
-        },
-        link: {
-            color: 'rgb(0, 0, 238)',
-        },
-    }))
+    function round(value, decimals) {
```

Comment:
This is tangential, but in one of the next sprints I think we should make a lab-specific labkey-util package, where we can begin to collect generic helper code, either general purpose, or labkey-specific

Reply:
Created a base issue #61

Reply:
Yes, that's probably the best way to do it. It also lets us manage all of our own dependencies -- we could have a separate package.json